test/unit/specs/renderer/services/formatting.spec.js

```diff
@@ -0,0 +1,82 @@
+    });
+  });
+
+  describe('formatDateShort', () => {
+    it('should properly format', () => {
+      expect(formatting.formatDateShort(1524752757)).to.eql('26-04');
+      expect(formatting.formatDateShort('1524752757')).to.eql('26-04');
+      expect(formatting.formatDateShort(null)).to.eql('--');
+      expect(formatting.formatDateShort(undefined)).to.be.eql('--');
+      expect(formatting.formatDateShort(null, 'default')).to.eql('default');
+      expect(formatting.formatDateShort(undefined, 'default')).to.be.eql('default');
+    });
+  });
+
+  describe('formatMoney', () => {
+    it('should properly format', () => {
+      expect(formatting.formatMoney(100000000.000001)).to.eql('$100,000,000.00');
+      expect(formatting.formatMoney('100000000.000001')).to.eql('$100,000,000.00');
+      expect(formatting.formatMoney(100000000.1)).to.eql('$100,000,000.10');
+      expect(formatting.formatMoney('100000000.1')).to.eql('$100,000,000.10');
+      expect(formatting.formatMoney(100000000.01)).to.eql('$100,000,000.01');
+      expect(formatting.formatMoney('100000000.01')).to.eql('$100,000,000.01');
+      expect(formatting.formatMoney(100000000)).to.eql('$100,000,000.00');
+      expect(formatting.formatMoney('100000000')).to.eql('$100,000,000.00');
+      expect(formatting.formatMoney(100000000, '^^')).to.eql('^^100,000,000.00');
+      expect(formatting.formatMoney('100000000', '^^')).to.eql('^^100,000,000.00');
+      expect(formatting.formatMoney(toBigNumber(100000000000000.0000000000001))).to.eql('$100,000,000,000,000.00');
+      expect(formatting.formatMoney(null)).to.eql('N/A');
+      expect(formatting.formatMoney(undefined)).to.be.eql('N/A');
+      expect(formatting.formatMoney(null, null, 'default')).to.eql('default');
+      expect(formatting.formatMoney(undefined, null, 'default')).to.be.eql('default');
+    });
+  });
+
+  describe('formatNumber', () => {
+    it('should properly format', () => {
+      expect(formatting.formatNumber(0.0000000000000001)).to.be.eql('0.0000000000000001');
+      expect(formatting.formatNumber('0.0000000000000001')).to.be.eql('0.0000000000000001');
+      expect(formatting.formatNumber(100000000)).to.be.eql('100,000,000');
+      expect(formatting.formatNumber('100000000')).to.be.eql('100,000,000');
+      expect(formatting.formatNumber(100000000, '0[.]0  ')).to.be.eql('100000000');
+      expect(formatting.formatNumber('100000000', '0[.]0  ')).to.be.eql('100000000');
+      expect(formatting.formatNumber(-100000000)).to.be.eql('-100,000,000');
+      expect(formatting.formatNumber('-100000000')).to.be.eql('-100,000,000');
+      expect(formatting.formatNumber(100000000.00000001)).to.be.eql('100,000,000.00000001');
+      expect(formatting.formatNumber('100000000.00000001')).to.be.eql('100,000,000.00000001');
+      expect(formatting.formatNumber(toBigNumber(100000000000000.0000000000001))).to.eql('100,000,000,000,000');
```

Comment:
Maybe a test that creates this big number from a string and then verified that the string it formats also contains the satoshi.